schemas/se00_data.fbs

```diff
@@ -0,0 +1,45 @@
+//Used to transmit fast sample environment data
```

Comment:
As mentioned in the group meeting, this description should be more generic, e.g. used for transmitting waveform data